CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ a release.
 ---
 
 ## [Unreleased]
+- unify namespace usage of doctrine ORM mappings in `Translatable` and `Translator` entities (#2702)
```

Comment:
Is this change fixing some bug, changing an existing behavior or adding a new feature?
If not, I think we should omit the changelog in this case.

Reply:
No, actually it isn't - I was just following the notes in the [Contributing](https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/CONTRIBUTING.md#changelog) file. So if you want to, i can remove it.

Reply:
I removed it ✅